### PR TITLE
1st draft of exchange c-chain integration

### PR DESCRIPTION
**Pull request by jpop32**

### CODE REVIEW

File: build/tutorials/exchanges/exchange-c-chain-integration.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Avalanche C-Chain Integration Guide
+
+## Overview
+
+The objective of this document is to provide a brief overview of how to integrate with the EVM-Compatible Avalanche C-Chain. For teams that already support ETH, supporting the C-Chain is as straightforward as spinning up an Avalanche node (which has the same API as go-ethereum) and populating Avalanche’s ChainID (43114) when constructing transactions.
```
Comment (patrick-ogrady) [inline build/tutorials/exchanges/exchange-c-chain-integration.md:5]: Can we add a link for "same API" to https://eth.wiki/json-rpc/API and https://geth.ethereum.org/docs/rpc/server

Comment (patrick-ogrady) [inline build/tutorials/exchanges/exchange-c-chain-integration.md:5]: Can we add a link for ChainID info in the docs?

File: build/tutorials/exchanges/exchange-c-chain-integration.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### Running Avalanche node
+
+If you want to build your node form source or include it in a docker image, reference the [AvalancheGo GitHub repository](https://github.com/ava-labs/avalanchego). To quickly get up and running, you can use the [node installation script](../nodes-and-staking/set-up-node-with-installer.md) that automates installing and updating avalanchego node as a systemd service on Linux, using prebuilt binaries.
+
+### Configuring Avalanche node
+
+All command line options available are described [here](../../references/command-line-interface.md), along with their default values.
+
+You can supply options on the command line, or use the config file, which is probably easier to work with. Use `—config-file=config.json` option, and then provide complete configuration in the `config.json` file, for example:
```
Comment (patrick-ogrady) [inline build/tutorials/exchanges/exchange-c-chain-integration.md:15]: "which is probably easier to work with" -> "which can be easier to work with when a lot of custom options are configured"

File: build/tutorials/exchanges/exchange-c-chain-integration.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+```json
+{
```
Comment (patrick-ogrady) [inline build/tutorials/exchanges/exchange-c-chain-integration.md:17]: I'd just include the `coreth-config` options here and prettify it:
```json
{
  "coreth-config": {
    "snowman-api-enabled": false,
    "coreth-admin-api-enabled": false,
    "net-api-enabled": true,
    "eth-api-enabled": true,
    "personal-api-enabled": true,
    "tx-pool-api-enabled": true,
    "debug-api-enabled": true,
    "web3-api-enabled": true,
    "local-txs-enabled": true
  }
}
```

We may want to disable personal by default too? And provide another config below with it enabled.

File: build/tutorials/exchanges/exchange-c-chain-integration.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+"db-dir": "./db/",
+"http-port": 9545,
+"log-level": "verbo",
+"log-dir": "./log/"
+}
+```
+
+### Interacting with the C-Chain
+
+Interacting with the C-Chain is identical as interacting with `go-ethereum`. You can find the reference material for C-Chain API [here](../../avalanchego-apis/contract-chain-c-chain-api.md).
```
Comment (patrick-ogrady) [inline build/tutorials/exchanges/exchange-c-chain-integration.md:39]: Add a link to `go-ethereum`

File: build/tutorials/exchanges/exchange-c-chain-integration.md (hunk 3 above)
Comment (patrick-ogrady) [reply build/tutorials/exchanges/exchange-c-chain-integration.md:17]: I think people will use this as the default if we aren't careful.

File: build/tutorials/exchanges/exchange-c-chain-integration.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+If you plan on extracting data from the C-Chain into your own systems using golang, we recommend using our custom [ethclient](https://github.com/ava-labs/coreth/tree/master/ethclient). The standard go-ethereum Ethereum client does not compute block hashes correctly (when you call `block.Hash()`) because it doesn't take into account the added `[ExtDataHash](https://github.com/ava-labs/coreth/blob/2c3cfac5f766ce5f32a2eddc43451bdb473b84f1/core/types/block.go#L98)` header field in Avalanche C-Chain blocks, which is used move AVAX between chains (X-Chain and P-Chain). You can read more about our multi-chain abstraction [here](../../../learn/platform-overview/README.md) (out of scope for a normal C-Chain integration).
+
+If you plan on reading JSON responses directly or use web3.js (doesn't recompute hash received over the wire) to extract on-chain transaction data/logs/receipts, you shouldn't have any issues!
+
+## Summary
+
+We hope that his document will enable you to quickly integrate with Avalance Network on the C-Chain. If you have any problems or questions, reach out to us either directly to our developers, or on our public [Discord](https://chat.avalabs.org/) server.
```
Comment (patrick-ogrady) [inline build/tutorials/exchanges/exchange-c-chain-integration.md:70]: I'd remove this: `We hope that his document will enable you to quickly integrate with Avalance Network on the C-Chain.`

File: build/tutorials/exchanges/exchange-c-chain-integration.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+### (Optional) Custom Golang SDK
+
+If you plan on extracting data from the C-Chain into your own systems using golang, we recommend using our custom [ethclient](https://github.com/ava-labs/coreth/tree/master/ethclient). The standard go-ethereum Ethereum client does not compute block hashes correctly (when you call `block.Hash()`) because it doesn't take into account the added `[ExtDataHash](https://github.com/ava-labs/coreth/blob/2c3cfac5f766ce5f32a2eddc43451bdb473b84f1/core/types/block.go#L98)` header field in Avalanche C-Chain blocks, which is used move AVAX between chains (X-Chain and P-Chain). You can read more about our multi-chain abstraction [here](../../../learn/platform-overview/README.md) (out of scope for a normal C-Chain integration).
+
+If you plan on reading JSON responses directly or use web3.js (doesn't recompute hash received over the wire) to extract on-chain transaction data/logs/receipts, you shouldn't have any issues!
+
+## Summary
```
Comment (patrick-ogrady) [inline build/tutorials/exchanges/exchange-c-chain-integration.md:68]: Change this to `Support`

File: build/tutorials/exchanges/exchange-c-chain-integration.md (hunk 1 above)
Comment (jpop32) [reply build/tutorials/exchanges/exchange-c-chain-integration.md:5]: We don't have it anywhere as a separate item to link to. It is noted in the C-Chain API docs to which we link below in 'Interacting with C-Chain' section.

**Review by cgcardona — APPROVED** [review]

**Pull request merged**